R/utils_general.R

```diff
@@ -14,3 +14,47 @@
            width = 6,
            scientific = FALSE)
 }
+
+#' @title Update documentation
```

Comment:
Can you add some details describing the need for this function fulfills? 

I see it creates a PDF of the help files. Is this something needed given that the package build already includes the PDF? 

I personally find the website to be a much much much better representation of the help files compared to the PDFs.

Reply:
I have moved it locally and see if we get the request for some documentation in pdf inside the package.

Reply:
FYI
- CRAN builds and ships a PDF document with every release. https://cran.r-project.org/web/packages/visR/visR.pdf
- You can build that manual anytime you'd like with `devtools::build_manual()`